.gitignore

```diff
@@ -3,3 +3,5 @@ npm-debug.log
 build
 .antwar
 
+.idea
```

Comment:
Nope, that's really bad practice. That would mean an almost infinitely growing list of `.DS_Store`, vim swap files, Sublime project files, etc. All the while the developer can add this to his personal `.gitignore` file so the project is independent :).